### DISCUSSION TITLE
GitHub Desktop squash showing 2 popups

### DISCUSSION THREAD

**Discussion by Idomo** [community/community#56684] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Select Feature Area

- [x] Issues
- [ ] Projects

### Body

*I'm sorry if this isn't the place to report about this kind of bugs, but I couldn't find another one.

#### Tools:
- GitHub Desktop Version: 3.2.3 (arm64, macOS)

#### Description:
- When squashing 2 commits together, there's a popup to confirm the action and edit the new commit's title and description.
- Since the last 2 update (not sure what version I had before this one), after the confirmation of this popup and completing the squash action, it showing another empty popup (that is really annoying to dismiss every time).
- See attached video:

https://github.com/community/community/assets/26777129/d5ed4119-7696-4ecf-9dcb-56e334420fb0

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Idomo ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by Idomo** [reply]
Thank you @queenofcorgis.
Please refer someone that may be able to solve the bug (if you can).

Thanks,
Ido.

**Comment by Idomo** [comment]
Still happens with version `3.2.6`

**Comment by mufthakherul** [comment]
If can't solve from above suggation reinstall application.

**Reply by Idomo** [reply]
Where do you see a suggestion above?

**Reply by Idomo** [reply]
@miraz090906 Except of deleting all of my settings, reinstalling didn't done anything...

**Reply by mufthakherul** [reply]
https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/squashing-commits-in-github-desktop

**Reply by mufthakherul** [reply]
find your problem

**Reply by Idomo** [reply]
I know what squash is and how to perform it, please watch the video in the original post to understand the bug.
